### ISSUE TITLE
ease use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.   warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)

### ISSUE THREAD

**Issue by umie0128** [pytorch/pytorch#58894]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

**Comment by gchanan** [comment]
Please include the case.
